apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -78,12 +78,6 @@ class GqlWithKnexLoadList {
             knexQuery.leftJoin(...join(idx))
         })
         knexQuery.whereIn('mainModel.id', ids)
-        if (offset) {
-            knexQuery.offset(offset)
-        }
-        if (limit) {
```

Comment:
Why limit was removed?

Reply:
Offset and limit was set for related tables, in case of loading non-first chunk caused to return no data for them.